src/main/java/core/basesyntax/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReaderServiceImpl implements ReaderService {
+    private static final int HEADER_ROW = 1;
+
+    @Override
+    public List<String> readData(String fileName) {
+        try {
+            return Files.readAllLines(Path.of(fileName)).stream()
+                    .skip(HEADER_ROW)
```

Comment:
let's skip 1st row in parser please
because now your reader cannot be reused to read other files